### PR TITLE
Adds waitress and waitress bonus

### LINKED ISSUE #5: Add waitress counter
Count the waitresses a player has, including the waitress bonus. The waitress bonus should be `#a17ad3`. The price logic will need to be updated. Each waitress should be +$3, the bonus adds +$2 per waitress. (i.e. +$5 for each waitress instead).
The counter should not be allowed below 0, and the upper limit should be 12 as that's the number of waitress cards in the game.

### PR DESCRIPTION
**Pull request by dan-snw**
Fixes #5 

## Changes

Adds a waitress and waitress bonus. Formatting not yet done hence a draft request.

### CODE REVIEW

File: package-lock.json
Comment (egnwd) [inline package-lock.json:1]: You’ll want to delete this and run yarn

File: src/Components/HouseDemandContext.tsx [hunk 1]
```diff
@@ -27,6 +27,10 @@ export type HouseDemand = {
   setHasGarden: React.Dispatch<React.SetStateAction<boolean>>;
   hasCFO: boolean;
   setHasCFO: React.Dispatch<React.SetStateAction<boolean>>;
+  hasFirstWaitress : boolean;
```
Comment (egnwd) [inline src/Components/HouseDemandContext.tsx]: First waitress is a milestone so should be included in that object

**Review by egnwd — CHANGES_REQUESTED** [review]
Looks good, just a couple of small tweaks

File: src/Components/HouseDemandContext.tsx (hunk 1 above)
Comment (dan-snw) [reply src/Components/HouseDemandContext.tsx]: Good point, it's resolved.

File: package-lock.json
Comment (dan-snw) [reply package-lock.json:1]: Oh sorry I missed this when rebasing, it's now deleted.

**Review by egnwd — APPROVED** [review]

**Pull request merged**